### DISCUSSION TITLE
Discussion while building

### DISCUSSION THREAD

**Discussion by Im-Madhur-Gupta** [iYojna/iYojana-mobile-app#2] [General]
Heyoo, we can use this space for discussions.

**Comment by Im-Madhur-Gupta** [comment]
Current tasks we are working on -

@mugdha273 -> user backend
@PrathakGarg -> scraping
@AkhileshManda -> flutter app -> text-to-speech, vice-versa and translation
@DhruvDave12 -> flutter app
@kalashshah -> admin web app
@Im-Madhur-Gupta -> bot service integ on backend
@RiyaParikh0112 -> OCR
@AnkurKumarShukla  -> document the recommendation system

**Comment by Im-Madhur-Gupta** [comment]
@all be ready for some evaluation. 🤩✨😌

**Reply by RiyaParikh0112** [reply]
yayyy!!! brush up speaking skills :) lets do some marketing XD

**Comment by AnkurKumarShukla** [comment]
Recomendataion system :  **2d array of  nlp passed ,session wise  data** -----> apriori algorithing -----> ranking in nlp based---->vectorise search result / search------->>

**Comment by RiyaParikh0112** [comment]
Gujrati OCR model Pickled!! 🚀

**Comment by mugdha273** [comment]
Backend hosted
https://iyojna-backend.herokuapp.com/

use https://iyojna-backend.herokuapp.com/swagger/ 
to try out all the apis

**Comment by mugdha273** [comment]
@frontend people: in register API, username password is hardcoded, pass anything to that. I've extracted the first part of email and set up as username and password is hardcoded for everyone.

**Comment by mugdha273** [comment]
APIs ready to use:
(1) Register

**Comment by RiyaParikh0112** [comment]
Additional Chatbot Todo:
Summarization
Vectorization : For further similarity

**Comment by RiyaParikh0112** [comment]
Jury Suggestion:
1. What if we are asking for someone else
2.  Post Survey after using chatbot

**Comment by AnkurKumarShukla** [comment]
![image](https://user-images.githubusercontent.com/80956033/195979938-4c6f5123-bd99-4121-829b-f0d39bfb9488.png)


Direct ocr the image click here : <br>
[click here ](https://onlineocrconverter.com/languages)<br>

OCR  api found
